internal/cluster/transition.go

```diff
@@ -600,6 +628,22 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 				sCluster.cluster.OpenshiftVersion, *sCluster.cluster.ID, sCluster.cluster.EmailDomain,
 				sCluster.cluster.InstallStartedAt)
 		}
+
+		// If the cluster just transitioned into the installing state then check whether or not we have ignored any validations.
+		if sCluster.srcState != swag.StringValue(sCluster.cluster.Status) && swag.StringValue(sCluster.cluster.Status) == models.ClusterStatusInstalling {
+			if HasIgnoredValidations(*sCluster.cluster) {
+				// Send an event to indicate that validations are being ignored.
+				eventgen.SendValidationsIgnoredEvent(params.ctx, th.eventsHandler, *sCluster.cluster.ID)
+				// Set a feature usage flag to indicate that validations are being ignored.
+				if usages, u_err := usage.Unmarshal(sCluster.cluster.FeatureUsage); u_err == nil {
+					th.setUsage(true, usage.ValidationsIgnored, nil, usages)
```

Comment:
Why is the setUsage called when moving to installing? Shouldn't it be set when moving to preparing ?

Reply:
Maybe... "preparing" is the first stage after the user starts the install right?

I'll look into that

Reply:
As discussed with you, I'm going to look at moving this to the "install" API call as this should greatly simplify things